cmd/flowrunner/testdata/two_questions.json

```diff
@@ -94,8 +105,7 @@
                     "router": {
                         "type": "switch",
                         "wait": {
-                            "type": "msg",
-                            "timeout": 600
```

Comment:
thanks to copy and pasting flow defs there were lots of cases of example flows with timeouts set but case or exit for them etc